src/lib/network_pool/batcher.ml

```diff
@@ -33,7 +35,9 @@ type ('init, 'partially_validated, 'result) t =
             fails to verify. *)
          [ `Init of 'init | `Partially_validated of 'partially_validated ] list
       -> [ `Valid of 'result
-         | `Invalid
+         | `Invalid_keys
```

Comment:
can we add list of public keys (associated with the party) to these constructors? It'll be difficult to debug given the errors can be from any party of a snapp txn

Reply:
Done.